scripts/update-version.sh

```diff
@@ -10,10 +10,17 @@ set -e
 
 NEW_VERSION="$1"
 
-echo "Updating package.json to new version: $NEW_VERSION"
-
+# Update new version in package.json file
+PACKAGE_JSON_SOURCE_FILE="package.json"
+echo "Updating package.json at: $PACKAGE_JSON_SOURCE_FILE to new version: $NEW_VERSION"
 npm version "$NEW_VERSION" --no-git-tag-version --allow-same-version
+echo "Done! Showing changes to confirm:"
+git diff $PACKAGE_JSON_SOURCE_FILE
 
-echo "Check package.json file. You should see version inside has been updated!"
-
-cat package.json
\ No newline at end of file
+# Update new version in customer_io_config.xml file
+SDK_CONFIG_XML_FILE="android/src/main/res/values/customer_io_config.xml"
+SDK_CONFIG_CLIENT_VERSION_KEY="customer_io_react_native_sdk_client_version"
+echo "Updating customer_io_config.xml at: $SDK_CONFIG_XML_FILE to new version: $NEW_VERSION"
+sd "<string name=\"$SDK_CONFIG_CLIENT_VERSION_KEY\">.*</string>" "<string name=\"$SDK_CONFIG_CLIENT_VERSION_KEY\">$NEW_VERSION</string>" $SDK_CONFIG_XML_FILE
```

Comment:
to use `sd` here, we need to ensure we install this dependency before calling it. Just make sure to verify that.

Reply:
Good call, it was missing, so I added it to the workflow. For now, I installed `sd` the same way we're doing for other repos, but I think there's an improvement needed, which we can address later together as part of overall CI improvements across all repos.